### PR TITLE
Feature/big sur and linux versions

### PR DESCRIPTION
**Pull request by remarkov**
add versions report for macOS Big Sur image and Linux image (beta)

### CODE REVIEW

File: content/specs/versions3.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+description: A list of tools available out-of-the-box on Codemagic build machines.
+title: macOS build machine specification (Xcode 12.5+)
+aliases:
+  - '../releases-and-versions/versions3'
+weight: 1
```
Comment (helinanever) [inline content/specs/versions3.md:6]: weight: 3

File: content/specs/versions3.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- tar
+- unzip
+- wget
+- yarn `1.22.10`
+- yq
+- zip
+
+## Xcode 12.5 (12E5244e)
+
+This is the Xcode version used by default when you select `latest` in build settings. Other available versions are listed [here](#other-xcode-versions).
```
Comment (helinanever) [inline content/specs/versions3.md:54]: Once we make this version available, it should be the edge, right? So I would change here "when you select 'edge'"

File: content/specs/versions3.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- Apple Watch Series 5 - 40mm
+- Apple Watch Series 5 - 44mm
+- Apple Watch SE - 40mm
+- Apple Watch SE - 44mm
+- Apple Watch Series 6 - 40mm
+- Apple Watch Series 6 - 44mm
+
+## Other Xcode versions
+
+- 12.4 (12D4e) `/Applications/Xcode-12.4.app`
```
Comment (helinanever) [inline content/specs/versions3.md:147]: Are you sure we have 12.4 available on Big Sur image?

File: content/specs/versions-linux.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+description: A list of tools available out-of-the-box on Codemagic build machines.
+title: Linux build machine specification
+aliases:
+  - '../releases-and-versions/versions-linux'
+weight: 1
```
Comment (helinanever) [inline content/specs/versions-linux.md:6]: weight: 4

File: content/specs/versions-linux.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+---
+description: A list of tools available out-of-the-box on Codemagic build machines.
+title: Linux build machine specification
```
Comment (helinanever) [inline content/specs/versions-linux.md:3]: I propose to add (beta) to the title

**Review by helinanever — COMMENTED** [review]
Added some comments to address

File: content/specs/versions3.md (hunk 1 above)
Comment (remarkov) [reply content/specs/versions3.md:6]: why 3? as I understand that would make it after xcode-11 specs and we will have them in order "xcode-12", "xcode-11", "xcode-12.5". I think we should leave 1 here and shift the others to be below this one so the order will be "xcode-12.5", "xcode-12", "xcode-11".

Comment (helinanever) [reply content/specs/versions3.md:6]: true, agree

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**